packages/field-label/README.md

```diff
@@ -77,34 +103,28 @@ import { FieldLabel } from '@spectrum-web-components/field-label';
 </sp-tab-panel>
 </sp-tabs>
 
-## Examples
-
-```html
-<sp-field-label for="lifestory">Life Story</sp-field-label>
-<sp-textfield placeholder="Enter your life story" id="lifestory"></sp-textfield>
-<sp-field-label for="birth-place">Birthplace</sp-field-label>
-<sp-picker id="birth-place">
-    <span slot="label">Choose a location:</span>
-    <sp-menu-item>Istanbul</sp-menu-item>
-    <sp-menu-item>London</sp-menu-item>
-    <sp-menu-item>Maputo</sp-menu-item>
-    <sp-menu-item>Melbuorne</sp-menu-item>
-    <sp-menu-item>New York</sp-menu-item>
-    <sp-menu-item>San Fransisco</sp-menu-item>
-    <sp-menu-item>Santiago</sp-menu-item>
-    <sp-menu-item>Tokyo</sp-menu-item>
-</sp-picker>
-```
+#### Label Position
 
-## Side Aligned
+Field labels can be positioned either on top of an input (default) or to the side of an input. The top position is recommended for most cases as it works better with long text, localization, and responsive layouts.
 
 Using the `side-aligned` attribute will display the `<sp-field-label>` element inline with surrounding elements and the `start` and `end` values outline the alignment of the label text in the width specified.
 
-### Start
+<sp-tabs selected="top" auto label="Label position options">
+<sp-tab value="top">Top (Default)</sp-tab>
+<sp-tab-panel value="top">
 
-Use `side-aligned="start"` to display the `<sp-field-label>` inline and to align the label text to the "start" of the flow of text:
+```html demo
+<sp-field-label for="country-top">Country</sp-field-label>
+<sp-textfield placeholder="Enter your country" id="country-top"></sp-textfield>
+```
 
-```html
+</sp-tab-panel>
+<sp-tab value="side-start">Side (Start Aligned)</sp-tab>
+<sp-tab-panel value="side-start">
+
+Using `side-aligned="start"` displays the `<sp-field-label>` inline and aligns the label text to the "start" of the flow of text:
```

Comment:
Use `side-aligned="start"` to display the `<sp-field-label>` inline and to align the label text to the "start" of the flow of text: